docs/sql/json.md

```diff
@@ -15,6 +15,10 @@ compare the runtime type with `T`.  If the types are identical or there
 is a natural conversion from the runtime type to `T`, the original value
 is returned.  Otherwise the `CAST` returns `NULL`.
 
+A value of type `VARIANT` that stores a `MAP` can be converted to a
+user-defined type.  Each name of a field of the user-defined type is
+used as to index into the map.
```

Comment:
```suggestion
used as an index into the map.
```

Reply:
Describe conversion rules: map key names must match struct field names, what happens with extra keys in the map that are not in the struct, what happens with missing fields?  What happens if conversion fails. How is case sensitivity handled?